theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -4,29 +4,29 @@ set -xe
 # This script sets up the PBuilder hooks to pull in our deb repositories while the package is building
 echo "--Apt Hook Setup"
 
-for pbuilderd in /etc/pbuilder/${os}*; do
-  # Cleanup any old hooks
-  sudo rm -f ${pbuilderd}/hooks/F60addforemanrepo
+pbuilder_hooksdir=/etc/pbuilder/${os}64/hooks
 
+# Cleanup any old hooks
+sudo rm -f ${pbuilder_hooksdir}/hooks/F60addforemanrepo
```

Comment:
(repeated a few times)
```suggestion
sudo rm -f ${pbuilder_hooksdir}/60addforemanrepo
```